packages/Adapters/POD/CMakeLists.txt

```diff
@@ -1,4 +1,4 @@
-TRIBITS_SUBPACKAGE(C_API)
+TRIBITS_SUBPACKAGE(PlainOldData)
```

Comment:
I actually think it will be more clear if we have two packages: C_API and Fortran_API. We will likely have a lot of C and Fortran work in our future so this would help. Also it separates the libraries we will construct - one library per subpackage. Even though there is only one Fortran file for now, I think this makes sense.